azext_iot/tests/iothub/configurations/test_config_modules_read.json

```diff
@@ -0,0 +1,98 @@
+{ 
+    "moduleTwins": [
+        {
+            "deviceId": "test-device-01",
```

Comment:
we should test with `$edgeAgent` and `$edgeHub` as well because they will always be part of an edge device

Reply:
sounds good, will update